_data/simple-icons.json

```diff
@@ -6600,6 +6600,11 @@
             "hex": "2D2E83",
             "source": "https://www.showpad.com/"
         },
+        {
+            "title": "Showtime",
+            "hex": "B10000",
+            "source": "https://www.cghnyc.com/work/project/showtime"
```

Comment:
Could you look to update the source to https://www.sho.com/ - just so it's a direct link to the brand's site for future reference?